### PR TITLE
Vampires will now repair their external limbs when resting in coffins

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
This only happened for internal organs, now it happens for limbs too
:cl:
 * tweak: Vampires will now also restore broken bones and stop bleeding when resting in coffins.

### CODE REVIEW

File: code/datums/gamemode/role/vampire_role.dm [hunk 1]
```diff
@@ -287,6 +287,11 @@
 				I.status &= ~ORGAN_BROKEN
 				I.status &= ~ORGAN_SPLINTED
 				I.status &= ~ORGAN_BLEEDING
+		for(var/datum/organ/external/O in H.organs)
+			if(O)
```
Comment (DamianX) [inline code/datums/gamemode/role/vampire_role.dm]: this will always be true

Comment (boy2mantwicethefam) [reply code/datums/gamemode/role/vampire_role.dm]: Nice

**Comment by Ratvar** [comment]
What about internal limbs?

**Comment by Kurfursten** [comment]
fix it

**Pull request merged**